### DISCUSSION TITLE
GitHub Project Filtering and Beta Tasklist

### DISCUSSION THREAD

**Discussion by Mexmorize** [community/community#53723] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

My team/org and I are loving the new functionality that GitHub Tasklist provide and the ability to better track hierarchy within our repos and work items.

I did notice something though, if you have multiple tasklists within the same issue, and if you group by `Tracked By` and filter by `label` on Projects, you can get two references to the same connection.

That might seem weird but I believe a 'fix' or 'feature' that would be good is the ability to group by `Tracked By` and then in the Filter, add the ability to have like

`tracked-by: label: "Epic"` which, in this example, would allow the user to only track children of issues that are labeled "Epic", ex: Features.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mtcA6** [comment]
I agree w/this, there's a problem with how the filtering is being done. It's filtering on issues in the grid but not considering the values of the tracking issue 

eg. i don't want to see completed items, be that issues (ie. tracked by) or 'epics' (ie. tracks)
I've filtered out the status of done, but as you can see from the screengrab below, it still pulls in the tracking issue that's been completed only because the issues it originally tracked are not completed. 
![image](https://github.com/community/community/assets/129431463/cc3963ee-b42c-4f1d-9d01-61ca5fb729e8)
